CFP and tickets

geokala opened the pull request.

Commit:
Initial CFP page

Commit:
Clean up links on cfp page

Commit:
Spell what correctly

Commit:
Add cfp closing date and correct link

meatballs commented:
Can we get rid of the posters card? I'm not sure we'll be doing that this year.

Commit:
Remove posters

geokala commented:
Removed.

Commit:
Don't include posters image

meatballs commented on src/call-for-proposals.md:
```diff
@@ -0,0 +1,75 @@
+  </figure>
+
+  <h3>Talks</h3>
+  <p><strong>Speak to the wonderful PyCon UK audience!</strong> Most talk slots are around 25 minutes, with some longer slots available in the schedule.</p>
```
Can we remove "Most talk slots are around 25 minutes, with some longer slots available in the schedule."

meatballs replied:
Perhaps replace with "Talk slots are 25 minutes"

geokala replied:
Updated.

meatballs commented on src/call-for-proposals.md:
```diff
@@ -0,0 +1,66 @@
+</div>
+
+<div class="box box_red">
+  <p>The deadline for submissions is <strong>17:00 (Europe/London)</strong> on <strong>Saturday, 2nd July, 2022 </strong>- but don't leave it until then. Selections will be made by SELECTDAY SELECTMONTHWORD.</p>
```
Let's make this "Selections will be made during the course of July."

geokala replied:
Updated.

Commit:
Add page on why you should submit a proposal

Commit:
Remove talk slot blurb

Commit:
Note when selections will be made

meatballs commented on src/call-for-proposals/why-you-should-submit-proposal.md:
```diff
@@ -0,0 +1,56 @@
+<div class="box box_blue">
+  <h3>Pycon UK has a large and enthusiastic audience</h3>
+  <p>Whatever it is you have to share, <strong>it's going to strike a chord</strong> with at least someone in our audience. Your contribution, however modest or specialised you think it might be, could be exactly what someone else has been waiting for.</p>
+  <p>A session at PyCon UK is an excellent way to <strong>raise interest in a project</strong> or <strong>find new collaborators</strong>, or to <strong>draw attention to a problem</strong>.</p>
```
nitpick: too many 'or's and 'to's

Instead:

<p>A session at PyCon UK is an excellent way to <strong>raise interest in a project</strong>, <strong>find new collaborators</strong>, or <strong>draw attention to a problem</strong>.</p>

geokala replied:
Updated.

meatballs commented on src/call-for-proposals/why-you-should-submit-proposal.md:
```diff
@@ -0,0 +1,56 @@
+  <h3>Pycon UK has a large and enthusiastic audience</h3>
+  <p>Whatever it is you have to share, <strong>it's going to strike a chord</strong> with at least someone in our audience. Your contribution, however modest or specialised you think it might be, could be exactly what someone else has been waiting for.</p>
+  <p>A session at PyCon UK is an excellent way to <strong>raise interest in a project</strong> or <strong>find new collaborators</strong>, or to <strong>draw attention to a problem</strong>.</p>
+  <p>Our audience are <strong>supportive, friendly and respectful</strong>, and they're there precisely because they want to hear about new things that they can become involved in and make a difference to.</p>
```
nitpick: there's only one audience, so it's singular - "Our audience is..." and "people are there precisely because..."

geokala replied:
Apparently I can't spell people. Update coming up, with god speling.

Commit:
Add proposal submission advice

Commit:
Update "Why..." page per @meatballs' suggestions

Commit:
Remove extra hyphen messing up code highlighting

Commit:
Dedupe style definitions

Commit:
Add tables to the styling

Commit:
Add ticket information

Commit:
Add page explaining ticket pricing approach

Commit:
Add tickets to header and somewhat sort links

Keeping Home at the start for easy navigation and 'Contact us' at the end for visibility.
However, everything else was getting a bit jumbled so now it's sorted by the link text.

Commit:
Add call to action to front page

meatballs commented on src/tickets.md:
```diff
@@ -0,0 +1,55 @@
+
+<div class="box box_yellow">
+  <h3>Please note</h3>
+  <p>As part of our return we are temporarily only offering tickets that cover the full conference. We are hoping to return to usual in 2023!</p>
```
let's get rid of 'temporarily' so people don't think we might change later this year.

geokala replied:
Removed.

meatballs commented on src/tickets.md:
```diff
@@ -0,0 +1,55 @@
+<div class="box box_yellow">
+  <h3>Please note</h3>
+  <p>As part of our return we are temporarily only offering tickets that cover the full conference. We are hoping to return to usual in 2023!</p>
+  <p>There are no specific childrens' tickets this year. However, ticket holders may bring two under-16s to the conference.</p>
```
up to two under-18s

geokala replied:
Updated.

Kinda amused at the thought of "all ticket holders must be accompanied by precisely two under-18s" now...

Commit:
Temporarily is accurate but misleading, say it not

Commit:
Correct accompanyied tickets note

Commit:
Make strong tags do something

Commit:
Put VAT number on new line

AnnJBarr commented:
That's great @geokala and @meatballs.
Could we just take the tickets link out for now and say 'Tickets will go on sale shortly' 
Then absolutely everything else is spot on

Commit:
Remove tickets link while finishing prep
